app/controllers/instructeurs/archives_controller.rb

```diff
@@ -13,12 +13,12 @@ def index
       @archives = current_instructeur.archives.where(procedure: procedure)
     end
 
-    def create
+    def create_archive
```

Comment:
titre du commit : ` remove remote form, uses get for creation instead `

ce commit me parait bizarre, je comprends pas pourquoi on utilise un GET pour une création de ressource. T'es sur qu'on veut pas un post ?

Reply:
supprimé